### DISCUSSION TITLE
Anyone know how to downgrade under chrome ver 106?

### DISCUSSION THREAD

**Discussion by Noob-ishh** [3kh0/ext-remover#288] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I tried all I could but it didn't work.

**Comment by SomePVZPlayerMatt** [comment, marked as answer]
DO NOT DOWNGRADE YOUR CHROMEBOOK IT WILL GET YOU SUSPENDED FROM SCHOOL

**Reply by rwg100** [reply]
![OIP](https://user-images.githubusercontent.com/118750743/204560687-7be1f2ca-3f40-4c06-9989-d6370f30d594.jpeg)

**Reply by Noob-ishh** [reply]
I WANNA DO IT

**Reply by SomePVZPlayerMatt** [reply]
please, dont.

**Reply by 3kh0** [reply]
![Screenshot 2022-11-29 13 20 22](https://user-images.githubusercontent.com/58097612/204614797-42e686cf-c803-49cd-98f7-15351552d695.png)

**Reply by Noob-ishh** [reply]
oh btw my school doesnt care

**Reply by pl4udible** [reply]
its a criminal offense lol

**Reply by nicolasog12** [reply]
what a skid

**Comment by ericshu6** [comment]
what if u just do webstore? the bookmarklet still works

**Comment by Aka-but-nice** [comment]
Now, don't say I didn't warn you but there is one way to downgrade chrome os, this is a foreign method that roamed earth years ago, press control-alt-t crosh will open up and type the command "rollback" and it will say press y or n,  Y for yes and N for no if you want to continue and may get your Chromebook taken away then press Y and it will powerwash your Chromebook and you will be on an old version of chrome!
